app/services/ccr/expenses_adapter.rb

```diff
@@ -72,7 +72,7 @@ def initialize(expense)
 
     def description
       parts = [(expense.location ||= expense.expense_type.name)]
```

Comment:
Not related to this PR, but this line looks wrong. If I understand correctly then, if `expense.location` is `nil` and at some point `expense.save` is called, then `expense.location` will get set to `expense.expense_type.name`. This looks like a potential bug that would be very difficult to track down.

Reply:
🤔  yes it does seem odd. although within this flow a claim has already been submitted, and the adaptor should not `save` it anywhere 🤞. 

The specs are no help either as we do not have an isolated `#description` test.
I think I would be happy to have it replaced with:`[(expense.location || expense.expense_type.name)]`

Reply:
This situation does no handle empty strings for location either, although I believe an empty string will never be saved except as nil in any event. Stats wise expenses on production with nil location is 11K+, but none with empty string `''`

Reply:
see this PR for fix https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/4272